### PR TITLE
fix build fails

### PR DESCRIPTION
**Pull request by VishnuSanal**
fixes the failing builds

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -148,7 +149,7 @@ subprojects {
             }
         }
         dependencies {
-            compileOnly 'com.github.pengrad:jdk9-deps:1.0'
+//            compileOnly 'com.github.pengrad:jdk9-deps:1.0'
```
Comment (VishnuSanal) [inline build.gradle:151]: @TranceLove what were this dependency used for? this causes the build failure. the builds pass even without this; can we safely remove it?

Comment (TranceLove) [reply build.gradle:151]: This was introduced when moving towards JDK 9 per #1941. Perhaps this is no longer needed when officially Android apps can build with JDK 11/17?

Comment (VishnuSanal) [reply build.gradle:151]: > This was introduced when moving towards JDK 9 per #1941. Perhaps this is no longer needed when officially Android apps can build with JDK 11/17?

Code complies & the tests pass -- shouldn't it be good? 🤔

**Comment by VishalNehra** [comment]
Needed these changes to fix build failures
https://github.com/TeamAmaze/AmazeFileManager/commit/958ec0a414d9c4de37fc5fc3e5271dea37ed209e

**Review by VishalNehra — APPROVED** [review]

**Comment by VishnuSanal** [comment]
> Needed these changes to fix build failures
> https://github.com/TeamAmaze/AmazeFileManager/commit/958ec0a414d9c4de37fc5fc3e5271dea37ed209e

What? Didn't get you. 🤔

**Pull request merged**